simple_parsing/wrappers/field_wrapper.py

```diff
@@ -1135,7 +1134,11 @@ def __init__(
             choices=choices,
             required=required,
             help=help,
-            metavar=metavar,
+            # TODO: Ideally, this should be depend on `nargs`:
+            # '?': `--a [bool]`
+            # 1: `--a bool` 
+            # '+': `--a bool [bool ...]`
+            metavar=metavar or "[bool]",
```

Comment:
Yeah that's a good point, although since this action is only used with plain `bool` fields, then it should probably be fine for now. This action isn't used when you have a `bool | None` field, or `list[bool]`, etc, so the `nargs` would only be different if it were set manually with the `field` function.

I guess we could fix this very minor issue later.

Reply:
Actually I think the `[]` should be automatically added here: https://github.com/python/cpython/blob/cb6ff54b01118e01c26758d2f57cf6da60277bd4/Lib/argparse.py#L595

Not sure why this isn't the case. Because this is such a minor issue. I'll be in favor of merging this now and investigate this later.

Reply:
Sure, fair enough